install.yaml

```diff
@@ -1,6 +1,29 @@
 name: tsh
```

Comment:
Should have #ddev-generated right?

Reply:
Ha, it sounds like yes? I assumed that install didn't get copied into the project so it didn't need it, but I guess everything does?